normalization/ufoNormalizer.py

```diff
@@ -684,7 +1215,17 @@ def _normalizeGlifComponentAttributesFormat2(element):
 
 def _normalizeGlifTransformation(element):
     """
-    TO DO: need doctests
+    >>> element = ET.fromstring('<component base="B" xScale="1" xyScale="0" yxScale="0" yScale="1" xOffset="0" yOffset="0"/>')
+    >>> _normalizeGlifTransformation(element)
+    {'yScale': '1', 'yxScale': '0', 'yOffset': '0', 'xyScale': '0', 'xScale': '1', 'xOffset': '0'}
```

Comment:
I just noticed that the values are being treated as strings. This could cause differences later on, and it's different from how other numeric values, such as **x** and **y**, are being handled. See e.g. https://github.com/unified-font-object/ufo-example-implementations/pull/2/files#diff-58f91eeff1e40665d77d77b78e62c039R936